FanSpeedControl/FanSpeedControl.js

```diff
@@ -102,7 +102,6 @@ const FanSpeedControlBase = kind({
 	},
 
 	defaultProps: {
-		icon: 'fan',
```

Comment:
I'm sorry but I didn't expect to change this. ;(
Since the default icon for FanSpeedControl is obviously "fan" so we don't need to add that every time if we just use FanSpeedControl. It will be very annoying.
If we set the `icon=""`, the icon won't be showing.
The problem was in the sampler and I got fix it from https://github.com/enactjs/agate/pull/586.
Thanks!